selfdrive/car/mazda/carcontroller.py

```diff
@@ -0,0 +1,44 @@
+from selfdrive.car.mazda import mazdacan
+from selfdrive.car.mazda.values import SteerLimitParams, Buttons
+from opendbc.can.packer import CANPacker
+from selfdrive.car import apply_std_steer_torque_limits
+
+class CarController():
+  def __init__(self, dbc_name, CP, VM):
+    self.apply_steer_last = 0
+    self.packer = CANPacker(dbc_name)
+    self.steer_rate_limited = False
+
+  def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, visual_alert,
```

Comment:
Remove unused args

Reply:
I just added those last night hoping to build on top of them as we go!

Reply:
I'd prefer to only include what's needed now. Easy enough to add later.